flag_set.go

```diff
@@ -20,6 +20,8 @@ type FlagSet interface {
 	StringArrayVarP(p *[]string, name, shorthand, usage string)
 }
 
+// PFlagSetProvider provides access to the underlying pflag.FlagSet; the FlagSet may be type asserted to this interface
+// WARNING: do not use this unless you know what you are doing!
```

Comment:
I don't have a ton of context on this repo, but could this comment share a little more context? Why specifically is this risky to use?

Reply:
Does the comment on the `FlagSet` interface provide that context? https://github.com/anchore/fangs/pull/19/files#diff-5b88ff640dd9d2bd19d9ba15569be994f5f764882064e43674d6b343fa7afb8fR9-R11